pkg/cmd/fixtures.go

```diff
@@ -54,14 +62,28 @@ func (fc *FixturesCmd) runFixturesCmd(cmd *cobra.Command, args []string) error {
 		afero.NewOsFs(),
 		apiKey,
 		fc.stripeAccount,
+		fc.skip,
 		stripe.DefaultAPIBaseURL,
 		args[0],
 	)
 	if err != nil {
 		return err
 	}
 
+	if len(fc.override) != 0 {
+		fixture.Override(fc.override)
+	}
+
+	if len(fc.add) != 0 {
+		fixture.Add(fc.add)
+	}
+
+	if len(fc.remove) != 0 {
+		fixture.Remove(fc.remove)
+	}
```

Comment:
I feel like this is a `fixtures.NewFixture` implementation detail and should be moved inside this function, instead of expecting the `FixturesCmd` package to know about this.

That would avoid this part being duplicated in the `trigger` package (`pkg/fixtures/triggers.go`)

Reply:
makes sense!